Classes/Emogrifier.php

```diff
@@ -1739,29 +1749,46 @@ private function getNodesToExclude(\DOMXPath $xPath)
      * @throws \InvalidArgumentException
      * @throws \RuntimeException
      */
-    public function handleXpathError($type, $message, $file, $line, array $context)
-    {
-        // Do not tie this conditional to debug mode as it causes unnecessary output. See further explanation in
-        // ::process() where this exception is caught.
-        if ($type === E_WARNING && isset($context['cssRule']['selector'])) {
-            throw new \InvalidArgumentException(
-                sprintf(
-                    '%1$s in selector >> %2$s << in %3$s on line %4$u',
-                    $message,
-                    $context['cssRule']['selector'],
-                    $file,
-                    $line
-                ),
-                1509279985
-            );
-        }
+    // @codingStandardsIgnoreLine
+    public function handleXpathQueryWarnings($type, $message, $file, $line, array $context)
+    {
+        if ($type === E_WARNING) {
```

Comment:
Wasn't the idea to remove this if? Otherwise, `$type` would not be unused, and we wouldn't need to silence PHP_CodeSniffer about the unused parameter.

Reply:
@oliverklee Of course it was... I was just rushing to submit... Removed the extra condition, moved phpcs comment to the end of the line, did the rebase. Please check. And sorry.

Reply:
Looks good. And there's no need to apologize. Forgetting/missing things just happens, that's normal, and that's what a second pair of eyes is for. :-)